include sections to index and main

potocki92 opened the pull request:
Added:
* $line-height-for-font-14: 14px to variables.scss
* partials to index.html
* scss to main.scss

potocki92 committed:
include sections to index and main

Added:
* $line-height-for-font-14: 14px to variables.scss
* partials to index.html
* scss to main.scss

MateuszCharysz commented on src/sass/utils/_variables.scss:
```diff
@@ -1,15 +1,19 @@
-    'green': #C2E297,
-    'grey-text': #907E82,
-    'light-grey': #E5E5E5,
+  'pink': #ffa5ba,
```
pink trzeba zamienić na pinky: #FFB8CA

MateuszCharysz commented on src/sass/main.scss:
```diff
@@ -1,12 +1,47 @@
+/* IMPORT SECTIONS
+============================================= */
+
+/* HEADER */
```
u mnie wystarczy importować header-menu - pozostałe pliki scss są importowane do header menu - wtedy jedyne co aktualizuje to swój plik scss a main będzie miał ten sam stan

potocki92 replied:
Aha, więc u Ciebie "_header-menu.scss" importuje resztę plików z head?

MateuszCharysz replied:
dokładnie - mniejszy ruch w main :)

MateuszCharysz reviewed (changes requested):
Co z pustymi plikami? aaaaa... już wiem jeśli ktoś robi na własnym branchu wtedy robi marge z main i powinno być ok.

potocki92 committed:
New commit

Changed 'pink' to 'pinky' in variables.scss
Deleted html from partials
Commented include from index and imports from main

MateuszCharysz reviewed (approved):
Działa
